config/global.ini.php

```diff
@@ -614,7 +614,9 @@
 record_statistics = 1
 
 ; length of a visit in seconds. If a visitor comes back on the website visit_standard_length seconds
-; after his last page view, it will be recorded as a new visit
+; after his last page view, it will be recorded as a new visit. In case you are using the Piwik JavaScript tracker to 
+; calculate the visit count correctly, make sure to call the method "setSessionCookieTimeout" eg 
+; `_paq.push(['setSessionCookieTimeout', timeoutInSeconds=1800000])`
```

Comment:
I copy / pasted example from piwik.js where it says `Set session cookie timeout (in seconds). Defaults to 30 minutes (timeout=1800000)` and `1800000` is actually the default value. Should be probably `1800` as it is multiplied by 1000 in the method.